wrkzcoin-nodes.json

```diff
@@ -42,6 +42,13 @@
       "ssl": false,
       "cache": false
     },
+    {
+      "name": "wrkz.pubnodes.com",
+      "url": "wrks.pubnodes.com",
```

Comment:
`wrkz.pubnodes.com` resolved but `wrks.pubnodes.com` not.

Reply:
LOL man...

yes its supposed to be wrkz :man_facepalming:

Reply:
Fixed...